test/splitting/invocations.stmt

```diff
@@ -248,9 +248,9 @@ someVeryLongTargetFunction(
   element
 ].someLongMethod();
 >>> do not split on "." when target is map
-{"key": "value", "another": "another value"}.someLongMethod();
+<String,String>{"key": "value", "another": "another value"}.someLongMethod();
```

Comment:
Is this change necessary? The expression in the expression statement isn't a map literal, it's a method invocation (`.someLongMethod()`) whose receiver is a map literal. Is that not valid?

Reply:
Correct. Although its valid as an expression, it's not valid as an expression statement.